docs/products/kafka/howto/monitor-logs-acl-failure.rst

```diff
@@ -0,0 +1,42 @@
+Monitor and alert logs for denied ACL
+=====================================
+
+Aiven for Apache Kafka supports the definition of users and access control lists (ACL) to allow and limit producer and consumer rights at topic level. You read more about ACLs permission in the :doc:`dedicated documentation <../concepts/acl>`.
```

Comment:
and here can we use the same "Aiven for Apache Kafka uses access control lists (ACL) and user definitions in order to establish individual rights to produce or consume a topic." as in other place?